exporter/awsemfexporter/cwlog_client.go

```diff
@@ -58,7 +58,7 @@ func newCloudWatchLogClient(svc cloudwatchlogsiface.CloudWatchLogsAPI, logger *z
 }
 
 // NewCloudWatchLogsClient create cloudWatchLogClient
-func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, startInfo component.ApplicationStartInfo, sess *session.Session) LogClient {
+func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, startInfo component.BinaryInfo, sess *session.Session) LogClient {
```

Comment:
```suggestion
func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, binaryInfo component.BinaryInfo, sess *session.Session) LogClient {
```

Reply:
Sure, I will rename all the variables from `startInfo` to `binaryInfo`